app/models/download.rb

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
 
+# Generated exports for download
 class Download < ApplicationRecord
+  has_many_attached :exports
```

Comment:
thoughts on renaming `:exports` to `:export_files` or `:files`? I was a bit confused at first when seeing this because a single 'export' in the context of SaraAlert generates one or multiple files to be downloaded, so I think something like `download.files.attach` would make more sense to me at first glance than `download.exports.attach` since a download does not contain 'exports' but rather the other way around

Reply:
Made the naming change